modules/deployment-model/src/main/java/org/mule/runtime/deployment/model/api/artifact/DescriptorLoaderRepositoryFactory.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.deployment.model.api.artifact;
+
+import org.mule.runtime.core.api.registry.SpiServiceRegistry;
+import org.mule.runtime.deployment.model.internal.artifact.ServiceRegistryDescriptorLoaderRepository;
+import org.mule.runtime.module.artifact.api.descriptor.DescriptorLoaderRepository;
+
+/**
+ * Factory to create instances of {@link DescriptorLoaderRepository}.
+ * 
+ * @since 4.5
+ */
+public class DescriptorLoaderRepositoryFactory {
+
+  private final SpiServiceRegistry serviceRegistry = new SpiServiceRegistry();
```

Comment:
I think this is more like a constant than a final instance field. Do you want to share this among all the DescriptorLoaderRepository's created by each factory but not for all the factories?